### ISSUE TITLE
ps_accounts the upgrade from 5.4.2 to 5.4.3 goes down the website given the error 503

### ISSUE THREAD

**Issue by Maofree** [PrestaShop/PrestaShop#32853]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hi
3 hours ago, I've updated the ps_accounts module to 5.4.3 from 5.4.2

the website goes down because the website
it was consuming all the resources of the vps and it was no longer visible online, after a few minutes of infinite loading, it returned the 503 error

renaming the module folder made the site visible again. now i'm loading the previous version of the module and see if it works as good as before. this module gives a little too many problems, even a month ago it sent me down two sites for server connection problems

bye

### Steps to reproduce

I don't know the steps

### PrestaShop version(s) where the bug happened

1.7.8.9

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

ps_accounts 5.4.3

**Comment by Maofree** [comment]
now I've set again the version 5.4.2 (just the folder without to reinstall it from admin panel) and all works well

there is an error on the 5.4.3

**Comment by florine2623** [comment]
Hello @maofree ,

I have ps_account v5.4.3 on PS 1.7.8.9.
<img width="1021" alt="Screenshot 2023-06-07 at 18 12 35" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/2bc0402d-d42f-4d28-b281-103bae13b40c">

I can log out and log in again without issue on BO. FO is displaying correctly as well.

Is there something more that I can try to reproduce your error ?

**Comment by Maofree** [comment]
Hi
I honestly can't give you much information. After installing the module, everything seemed fine, then after 10-15 minutes the company warned me that the site was giving a 503 error.
then I called the hosting and they told me that the site was consuming all the resources of the VPS. So I thought about the latest changes made and I remembered that I had recently updated ps_accounts and that 1 month ago this module created problems with two other sites, so I renamed the module folder, to check if it was his fault and the site is it worked again, so then I put the previous version back and there were no more errors
